app/assets/javascripts/documents.js

```diff
@@ -0,0 +1,46 @@
+console.log("ready");
+function sortContent(){
```

Comment:
This doesn't sort, it inits the sorting functionnality. So rename it to `initDocumentsSorting`
Also, since this is for the documents index, rename the file to `documents/index.js`